src/app/homepage/wallet-status/wallet-status.component.ts

```diff
@@ -98,7 +98,7 @@ export class WalletStatusComponent implements OnInit, OnDestroy {
       command = [
         new RpcSend('walletpassphrase', [
           this.stakingForm.value.stakingPassword,
-          999999999999999999,
+          99999999,
```

Comment:
You should actually be able to use the value `0` and it will remain unlocked for staking infinitely.